style.css

```diff
@@ -0,0 +1,342 @@
+body{
+	background-image: url(assets/style/concrete-texture-large.jpg);
+	background-repeat: no-repeat;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	font-family: 'Mitr', sans-serif;
+}
+header{
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	justify-content: space-between;
+	background-color: rgba(186, 171, 130, .8);
+	border: 3px solid rgba(166, 43, 39, 1);
+	box-sizing: border-box;
+
+	width: 1000px;
+	height: 50px;
+	border-radius: 10px;
+	padding: 20px;
+	margin: 10px;
+}
+main{
+	display: flex;
+	flex-direction: row;
+	align-items: flex-start;
+	justify-content: space-around;
+	background-color: rgba(32, 97, 204, .6);
+	border: 3px solid rgba(180, 35, 35, 1);
+	border-radius: 10px;
+	padding: 10px;
+	width: 1000px;
+	margin: 10px;
+
+	box-sizing: border-box;
+}
+label{
+	margin: 5px;
+	font-size: 20px;
+	color: white;
+	padding: 2px;
+
+}
+input{
+	background-color: rgba(59, 91, 105, .8);
+	border: 2px solid	rgba(5, 171, 101, 1);
+	color: white;
+	margin: 5px;
+	height: 30px;
+	width: 250px;
+	border-radius: 5px;
+	font-size: 20px;
+	padding: 2px;
+	box-sizing: border-box;
+}
+select{
+	background-color: rgba(59, 91, 105, .8);
+	border: 2px solid	rgba(5, 171, 101, 1);
+	color: white;
+	margin: 10px;
+	height: 30px;
+	width: 250px;
+	border-radius: 5px;
+	font-size: 20px;
+	padding: 2px;
+	box-sizing: border-box;
+
+}
+a{
+	color: white;
+	text-decoration: none;
+}
+a:hover{
+	color: rgba(5, 171, 101, 1);
+}
+#connect{
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	font-size: 20px;
+	width: 100px;
+}
+#githubLink{
+	height: 35px;
+	width: 35px;
+
+}
+#github{
+	width: 35px;
+	height: 35px;
+
+}
+#navbar{
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	justify-content: space-around;
+
+	background-color: rgba(59, 91, 105, .8);
+	border: 2px solid	rgba(5, 171, 101, 1);
+
+	height: 30px;
+	width: 500px;
+	margin: 10px;
+	border-radius: 10px;
+}
+#login{
+	color: black;
+	margin: 5px;
+	font-size: 20px;
+	padding: 2px;
+	box-sizing: border-box;
+}
+#landingMain{
+	display: flex;
+	flex-direction: row;
+	align-items: flex-start;
+	justify-content: space-around;
+	background-color: rgba(32, 97, 204, .6);
+	border: 3px solid rgba(180, 35, 35, 1);
+	border-radius: 10px;
+	padding: 10px;
+	width: 1000px;
+	margin: 10px;
+}
+#genInfo{
+	color: white;
+	width: 500px
```

Comment:
This one isn't acting correct for you because you forgot a semi-colon after `500px`